examples/source/1.components/amp-web-push.html

```diff
@@ -32,7 +32,7 @@
     <!--
     The `amp-web-push` component requires two additional HTML files as well as an amp-web-push compatible Service Worker. These three files form the configuration described below.
     -->
-    <amp-web-push id="amp-web-push" layout="nodisplay" helper-iframe-url="<% hosts.backend %>/webpush/amp-web-push-helper-frame.html" permission-dialog-url="<% hosts.backend %>/webpush/amp-web-push-permission-dialog.html" service-worker-url="<% hosts.backend %>/sw.js">
+    <amp-web-push id="amp-web-push" layout="nodisplay" helper-iframe-url="<% hosts.platform %>/static/samples/webpush/amp-web-push-helper-frame.html" permission-dialog-url="<% hosts.platform %>/static/samples/webpush/amp-web-push-permission-dialog.html" service-worker-url="<% hosts.backend %>/serviceworker.js">
```

Comment:
This won't work: <% hosts.backend %>/serviceworker.js. Instead, this sample should have it's own directory in which it contains it's own serviceworker + the other static files.